src/trio/_tests/test_util.py

```diff
@@ -274,6 +274,7 @@ def test_fixup_module_metadata() -> None:
     mod.SomeClass().method()
 
 
+@slow  # 1-2 seconds
```

Comment:
Why is this slow? I don't see any sleeps or whatever -- is our exception handling logic really slow?

Reply:
I have removed that decorator. I run that test alone a lot, and it's usually 1-2 seconds

it's a long test, that checks a lot of things in sequence, and as pytest.raises docs mention, functionality that records information for excinfo could make garbage collection slower (perhaps it's more pronounced in recent python versions that made changes to how gc/obj ref counts work in python, recent I mean > 3.9 )